tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
The clientside tsconfig is now in grumbler-scripts so this can be

```
{
  "extends": "@krakenjs/typescript-config-grumbler/tsconfig.json"
}
```